### PR TITLE
Update to .NET Core 2.0

### PR DESCRIPTION
**Pull request by stefanone91**

### CODE REVIEW

**Review by Che4ter — APPROVED** [review]

**Review by jackweldon — APPROVED** [review]

**Review by tornike87 — APPROVED** [review]

**Review by neb1neuron — APPROVED** [review]

**Review by neb1neuron — APPROVED** [review]

File: README [hunk 1]
```diff
@@ -4,7 +4,7 @@ Usage:
 
 public ActionResult PrintIndex()
 {
-    return new ActionAsPdf("Index", new { name = "Giorgio" }) { FileName = "Test.pdf" };
+    return new ActionAsPdf("Index", new { name = "Giorgio" }) { FileName = "Test.pdf" });
```
Comment (neb1neuron) [inline README:7]: This shouldn't be here

File: README.markdown [hunk 2]
```diff
@@ -5,7 +5,7 @@
 ```csharp
 public ActionResult PrintIndex()
 {
-    return new ActionAsPdf("Index", new { name = "Giorgio" }) { FileName = "Test.pdf" };
+    return new ActionAsPdf("Index", new { name = "Giorgio" }) { FileName = "Test.pdf" });
```
Comment (neb1neuron) [inline README.markdown:8]: Neither this line.

File: Rotativa.NetCore/Rotativa.NetCore.csproj [hunk 3]
```diff
@@ -28,27 +28,26 @@
...
     <CodeAnalysisRuleSet>..\stylecop.ruleset</CodeAnalysisRuleSet>
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <Description>A port of the Rotativa library for the ASP.NET Core library.</Description>
-    <PackageReleaseNotes>This is the initial version of the port and might have some bugs as it is not fully tested.</PackageReleaseNotes>
+    <Description>A port of the Rotativa library for the ASP.NET Core library, compatible with .NET Core 2.0.</Description>
+    <PackageReleaseNotes>Porting for .NET Core 2.0</PackageReleaseNotes>
     <Copyright>Copyright 2017</Copyright>
-    <Authors>Alexandru Munteanu</Authors>
-    <Company />
+    <Authors>Stefano Baruzzo</Authors>
```
Comment (neb1neuron) [inline Rotativa.NetCore/Rotativa.NetCore.csproj:40]: At least have the decency to append your name to the original author if you only changed the package version. Also you have a conflict. Please resolve that.

**Comment by paulbly** [comment]
How do i setup the configuration as the previous was RotativaConfiguration.Setup(env);

**Pull request closed without merging**